### DISCUSSION TITLE
Welcome to smplayer Discussions!

### DISCUSSION THREAD

**Discussion by smplayer-dev** [smplayer-dev/smplayer#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cootmaster** [comment]
hello , i notice we moved to github...  what  happened to old forum?

**Reply by smplayer-dev** [reply]
The old forum is here: http://old-forum.smplayer.info
You can use it to look for old posts, but posting new posts won't work,

**Reply by dandv** [reply]
Please set up a domain CNAME or some other sort of redirect to the old forum. There's a wealth of information there that's now [broken links](https://github.com/smplayer-dev/smplayer/discussions/9#discussioncomment-515198).

**Comment by Cootmaster** [comment]
why spit the  forum  and only allow   new posts here ya old forum breaks the postings

**Reply by smplayer-dev** [reply]
The server updated php or something and the forum got broken.  I don't really have time to try to fix it so...

**Comment by michaelbr51** [comment]
How to display Chinese char in SMPlayer?
I have a subtitle in Chinese, whey I tried to display it in SMPlayer it showed garbages, but Aegisub showed OK, I'm using MX Linux 19.3 and SMPlayer 21.1.0 (revision 9475), I've attached the subtitle if needed.
[Assembly.gb.srt.txt](https://github.com/smplayer-dev/smplayer/files/6163329/Assembly.gb.srt.txt)
ps: 1) I think my Linux is configured properly, I can see Chinese char in Brave browser. I've also installed Chinese char package in Linux. It should be a configuration problem but can't find it.
2) I changed the file name from Assembly.gb.srt to Assembly.gb.srt.txt, so I can add it to this comment.

Thanks

**Reply by smplayer-dev** [reply]
Try selecting "Simplified Chinese" in preferences -> subtitles -> Default subtitle encoding.

**Reply by dandv** [reply]
@michaelbr51: this question should be a separate topic. In the future, please start one.
This topic is just the intro for GitHub discussions.

**Comment by VLM-TechEd** [comment]
I want to start a new topic, but I don't see how. I don't see a "New topic" link or similar.

**Reply by egfontforgeA** [reply]
I have the same problem ? It's not obvious.

**Comment by VLM-TechEd** [comment]
Found a solution. 

 

MENU                                                Subtitles

            MENU ITEM                                      Download Subtitles from OpenSubtitles.com

                        “Find Subtitles” DIALOG                 Options button

 

THERE you can enter your OpenSubtitles.COM credentials. I did not try my .org credentials, though I am a VIP member.

 

I hope this helps!
